### ISSUE TITLE
R015a for VE11C multi band acceleration factor and other options not showing up

### ISSUE THREAD

**Issue by bedepue** [CMRR-C2P/MB#172]
Upgraded to VE11C on a Siemens SKYRA. Downloaded and installed R015a by disabling McAfee Embedded Control. The installer seems to flash quickly and then nothing. Now old paradigms using multi-band cmrr_mbep2d_* have no ability to select multi band acceleration factor and other options simply are missing. Any help would be appreciated.

**Comment by eauerbach** [comment]
If you try to run the installer .bat file directly (from the batch/ subdirectory) it will not work. You have to run the installer .exe located in the root directory of the sequence archive.

If you are able to open old protocols at all, then you must have successfully installed the sequences somehow. However, If you can not select multi-band acceleration, then the most likely thing is that you do not have the Siemens SMS license, which is required sinc VE11C.

**Comment by bedepue** [comment]
Thanks so much, Eddie. How do I get the SMS license, just contact them? Sorry for the naivety, I haven’t done much with sequences, for the exception of the MB.
Brendan

Brendan E. Depue, Ph.D.
Assistant Professor
Department of Psychological and Brain Sciences
Affiliated Professor
Department of Anatomical Sciences and Neurobiology
Endowed Chair of Behavioral Brain Imaging
Director of (N)euro (I)maging (L)aboratory of (C)ognitive, (A)ffective and (M)otoric (P)rocesses
NILCAMP: http://www.nilcamp.net
University of Louisville, Louisville, KY 40292
Phone: 502-852-6482

From: Eddie Auerbach
Reply-To: CMRR-C2P/MB
Date: Monday, September 18, 2017 at 10:02 AM
To: CMRR-C2P/MB
Cc: bedepue, Author
Subject: Re: [CMRR-C2P/MB] R015a for VE11C multi band acceleration factor and other options not showing up (#172)

SMS

**Comment by eauerbach** [comment]
If you contact your sales people they should be able to get you a demo license fairly quickly, but eventually you will have to buy it.

**Comment by bedepue** [comment]
Hi Eddie-
I emailed awhile back concerning the MB acceleration factor and other options not showing up after upgrading to VE11C and then installing R015a on a Skyra. You suggested that it was likely due to the SMS license not being installed.

Siemens has installed the SMS license, but unfortunately I keep getting the same failure to prepare reconstruction. I attach an images of the error. Full shut down and restart.

I know our MARS computer has no GPU, but is the latest 720 or 740 something or other model. I also looked through the logs, but found nothing. I imagine it really doesn’t have to do with reconstruction, as it doesn’t even start the scan or even if MB acc factor = 1. I tried the fix that someone else had posted about a broken symlink in support request #159. However, I assume that it’s the updated R015a is fixed because there was no symlink found.

I am not sure if it’s an installation problem, when I start the CMRR installer (.exe) it flashes and then nothing (Embedded control off), perhaps because it’s already installed? I can’t find a way to uninstall and reinstall, however I’m not sure if that would help anyway.

Any info is appreciated. Thanks,
Brendan

Brendan E. Depue, Ph.D.
Assistant Professor
Department of Psychological and Brain Sciences
Affiliated Professor
Department of Anatomical Sciences and Neurobiology
Endowed Chair of Behavioral Brain Imaging
Director of (N)euro (I)maging (L)aboratory of (C)ognitive, (A)ffective and (M)otoric (P)rocesses
NILCAMP: http://www.nilcamp.net<http://www.nilcamp.net/>
University of Louisville, Louisville, KY 40292
Phone: 502-852-6482


From: Brendan Depue
Date: Monday, September 18, 2017 at 10:18 AM
To: CMRR-C2P/MB
Subject: Re: [CMRR-C2P/MB] R015a for VE11C multi band acceleration factor and other options not showing up (#172)

Thanks so much, Eddie. How do I get the SMS license, just contact them? Sorry for the naivety, I haven’t done much with sequences, for the exception of the MB.
Brendan

Brendan E. Depue, Ph.D.
Assistant Professor
Department of Psychological and Brain Sciences
Affiliated Professor
Department of Anatomical Sciences and Neurobiology
Endowed Chair of Behavioral Brain Imaging
Director of (N)euro (I)maging (L)aboratory of (C)ognitive, (A)ffective and (M)otoric (P)rocesses
NILCAMP: http://www.nilcamp.net
University of Louisville, Louisville, KY 40292
Phone: 502-852-6482

From: Eddie Auerbach
Reply-To: CMRR-C2P/MB
Date: Monday, September 18, 2017 at 10:02 AM
To: CMRR-C2P/MB
Cc: bedepue, Author
Subject: Re: [CMRR-C2P/MB] R015a for VE11C multi band acceleration factor and other options not showing up (#172)

SMS
